libexec/tfenv-install

```diff
@@ -63,7 +63,8 @@ done;
 declare requested="${1:-""}";
 
 log debug "Resolving version with: tfenv-resolve-version ${requested}";
-declare resolved="$(tfenv-resolve-version ${requested})";
+declare resolved;
+resolved="$(tfenv-resolve-version ${requested})" || early_death "Failed to resolve ${requested} version";
```

Comment:
This should use:

```
|| log 'error' "Failed to resolve ${requested} version";
```

as the logging library has now been loaded. early_death is only for pre-logging initialisation

Reply:
Thanks for noticing, I've now made the change